src/config/index.ts

```diff
@@ -1,20 +1,23 @@
 import fs, { readFileSync } from 'node:fs';
 import { join } from 'node:path';
 
-import { go } from '@api3/promise-utils';
+import { goSync } from '@api3/promise-utils';
 import dotenv from 'dotenv';
 
 import { configSchema } from './schema';
 import { interpolateSecrets, parseSecrets } from './utils';
 
-export const loadConfig = async () => {
-  const configPath = join(__dirname, '../../config');
-  const rawSecrets = dotenv.parse(readFileSync(join(configPath, 'secrets.env'), 'utf8'));
+export const loadConfig = (
+  configPath = join(__dirname, '../../config'),
```

Comment:
Do we need these parameters? I think the implementation works even when the values are hadcoded.